network-api/networkapi/wagtailpages/pagemodels/customblocks/video_block.py

```diff
@@ -16,6 +18,31 @@ class VideoBlock(blocks.StructBlock):
         required=False,
         help_text='Optional URL for caption to link to.'
     )
+    wide_video = blocks.BooleanBlock(
```

Comment:
The update to the video_block model, as well as some validation to make sure the user does not pick wide and full width at the same time